daemon/state.go

```diff
@@ -42,6 +44,14 @@ func (s *State) String() string {
 		return fmt.Sprintf("Up %s", units.HumanDuration(time.Now().UTC().Sub(s.StartedAt)))
 	}
 
+	if s.removalInProgress {
+		return "Removal In Progress"
+	}
+
+	if s.Dead {
```

Comment:
Hm...., I don't understand this comment. I am first checking for Removal In Progress and dispalying that. Yes a container could be dead as well as removal is in progress. In that case I think it makes sense to display "Removal In Progress".

If that's not set, then I check for "Dead" state and display "dead". So where is the redundant check?